hoomd/mpcd/module.cc

```diff
@@ -86,7 +20,82 @@ namespace mpcd
  */
 namespace detail
     {
-    };
+void export_ATCollisionMethod(pybind11::module&);
```

Comment:
Nice! This type of change can significantly reduce re-compilation time when developing.